drivers/pwm/Kconfig.mcux

```diff
@@ -2,10 +2,21 @@
 
 # Copyright (c) 2019 Linaro Limited.
 # SPDX-License-Identifier: Apache-2.0
+# Copyright 2022 NXP
 
 config PWM_MCUX
 	bool "MCUX PWM driver"
 	default y
 	depends on DT_HAS_NXP_IMX_PWM_ENABLED
 	help
 	  Enable mcux pwm driver.
+
+if PWM_MCUX
+
+config PWM_WAITEN_FLAG
```

Comment:
should probably be `PWM_MCUX_WAITEN_FLAG`

Reply:
Question, do you want me to keep the if endif guards for the PWM_MCUX if I change this config to PWM_MCUX_WAITEN_FLAG?